go/common/types.go

```diff
@@ -26,9 +26,10 @@ type (
 
 	EncryptedParamsGetBalance   []byte // The params for an RPC getBalance request, as a JSON object encrypted with the public key of the enclave.
 	EncryptedParamsCall         []byte // As above, but for an RPC call request.
+	EncryptedParamsGetTxByHash  []byte // As above, but for an RPC getTransactionByHash request.
 	EncryptedParamsGetTxReceipt []byte // As above, but for an RPC getTransactionReceipt request.
+	EncryptedParamsLogs         []byte // As above, but for an RPC logs subscription request.
```

Comment:
Maybe `EncryptedParamsSubscribeLogs` or `EncryptedParamsLogsSubscription` - I know they're a bit long but it's useful when looking at these methods to be reminded it's a subscription and the the other Params types have a verb in them.

Reply:
Fixed.